### DISCUSSION TITLE
[Question] How do you import an pre existing Isaac Sim extension into Isaac lab?

### DISCUSSION THREAD

**Discussion by SirensAI** [isaac-sim/IsaacLab#1542] [Q&A] [answered]
Hey all,

I have already completed an extension in isaac sim, I was thinking the best way was to build out components in Sim then use the extension in lab, not sure how I can add these to my environment cfg?

thanks

**Comment by RandomOakForest** [comment, marked as answer]
Thank you for posting this question. This should be rather straight forward, you probably only need to be sure your extension is _enabled_. Look for example [how this was done for the Cloner extension](https://docs.omniverse.nvidia.com/isaacsim/latest/isaac_lab_tutorials/tutorial_cloner.html). Also, see as an example [how other learning libraries can be installed in your Python environment](https://isaac-sim.github.io/IsaacLab/main/source/how-to/add_own_library.html). Let us know if you need further help.

**Reply by Gary-Yifan-Zhang** [reply]
@RandomOakForest 
Hi! The original link you mentioned for the Cloner extension now returns 404.
Could you please update it to the latest working URL or point us to the replacement tutorial? Thank you for your advice.
